spread.yaml

```diff
@@ -122,11 +124,19 @@ suites:
 
   docs/reference/code/:
     summary: tests reference code from the docs
+    systems:
+    - ubuntu-22.04-64
+
+  tests/spread/rockcraft/:
+    summary: tests for rockcraft core functionality, independent of host system
+    systems:
+    - ubuntu-22.04-64
 
-  tests/spread/general/:
-    summary: tests for rockcraft core functionality
+  tests/spread/all-systems/:
```

Comment:
all systems being the suite and multiple being in the summary seems contradictory, let's rename this to the agreed "rockcraft" or "rockcraft-itself" (this is what I was referring to fwiw, the vanity name for the launchpad project on launchpad https://bugs.launchpad.net/launchpad)

Reply:
Can you elaborate on why you think `itself` is appropriate? From what I understand, it makes sense in launchpad's case because it is literally using launchpad to host launchpad. In rockcraft's case there is no "inception" going on - there are tests for rockcraft functionality, which should be independent of system, and tests that we explicitly want to run on multiple hosts, largely to run the generated rock.
I renamed the existing "general" suite to "rockcraft" for the tests that are host-independent. Would "multiple-systems" be better than "all-systems"?

Reply:
maybe call it alternative-systems or foreign-systems, or just foreign

Reply:
the rockcraft itself is mostly perspective, does rockcraf work on fedora is a quesion about rockcraft itself working on fedora and not so much about testing the features of rockcraft

Reply:
I'm fine with `foreign` (updated)